src/Models/generator.ts

```diff
@@ -0,0 +1,12 @@
+import { generateModelsAsync } from '@kentico/kontent-model-generator';
+
+generateModelsAsync({
+  sdkType: 'delivery',
+  projectId: 'da5abe9f-fdad-4168-97cd-b3464be2ccb9',
+  addTimestamp: true,
+  elementResolver: 'camelCase',
+}).then(() => {
+  console.log("ok")
+})
+
+export {}
```

Comment:
We shou export the metod and define this into the script in package,json or remove the unused functionality.

Reply:
[9ede5d4](https://github.com/Kentico/kontent-sample-app-react/pull/206/commits/9ede5d4e8127366824a28d497840cf0cf5e1cc2e)